packages/wdio-browserstack-service/src/service.js

```diff
@@ -27,8 +29,11 @@ export default class BrowserstackService {
     }
 
     afterTest(test) {
+        this.fullTitle = test.parent + ' - ' + test.title
+
         if (!test.passed) {
             this.failures++
+            this.failReason = test.error.message
```

Comment:
What if this is undefined?

Reply:
added check if undefined.
it should not be undefined unless the test passed